geode-management/src/main/java/org/apache/geode/management/api/EntityInfo.java

```diff
@@ -39,13 +40,13 @@
   private String id;
   @JsonInclude
   @JsonProperty
-  private List<EntityGroupInfo<T, R>> configurationByGroup = new ArrayList<>();
+  private List<EntityGroupInfo<T, R>> groups = new ArrayList<>();
 
   public EntityInfo() {}
 
-  public EntityInfo(String id, List<EntityGroupInfo<T, R>> configurationByGroup) {
+  public EntityInfo(String id, List<EntityGroupInfo<T, R>> groups) {
```

Comment:
Would `groupInfo` be a better name for this field (and the corresponding getter and setter)?

Reply:
I think "id" and "groups" seem to go together in this class. I would prefer to make the keys of the json short.